Lib/unittest/test/test_program.py

```diff
@@ -409,6 +409,15 @@ def testParseArgsAbsolutePathsThatCannotBeConverted(self):
         # for invalid filenames should we raise a useful error rather than
         # leaving the current error message (import of filename fails) in place?
 
+    def testParseArgsSelectedTestNames(self):
```

Comment:
Thank you for adding these tests. Do you think you could also create a small functional test? There is an example of such a test (using `subprocess`) in `Test_TextTestRunner.test_warnings`.

Reply:
I added a functional test but I'm not sure if that's the right place to put it.